src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/LaboratoryInvestigationResult.java

```diff
@@ -0,0 +1,120 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+import java.math.BigDecimal;
+
+/**
+ * Example: RSL+NV+11.9:7++:::ng/mL+HI'
+ */
+@Getter
+@Builder
+@AllArgsConstructor
+public class LaboratoryInvestigationResult extends Segment {
+
+    private static final String KEY = "RSL";
+    private static final String QUALIFIER = "NV";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+
+    private static final int MEASUREMENT_UNIT_SECTION = 4;
+    private static final int MEASUREMENT_UNIT_INDEX = 3;
+    private static final int DEVIATING_RESULT_INDICATOR_INDEX = 5;
+
+    private final BigDecimal measurementValue;
+    private final MeasurementValueComparator measurementValueComparator;
+    private final String measurementUnit;
+    private final DeviatingResultIndicator deviatingResultIndicator;
+
+    public static LaboratoryInvestigationResult fromString(String edifactString) {
+        if (!edifactString.startsWith(LaboratoryInvestigationResult.KEY_QUALIFIER)) {
```

Comment:
Same as in LaboratoryInvestigation :)

```suggestion
        if (!edifactString.startsWith(KEY_QUALIFIER)) {
```

Reply:
I see we do this in a number of other classes. I've made the same change in them